12.0

jad-odoo opened the pull request.

jad-odoo committed:
Add module defer_fields_computation

jad-odoo committed:
Change admin uid in module import_metadata

tfrancoi commented on defer_fields_computation/models/model.py:
```diff
@@ -0,0 +1,16 @@
+    _inherit = 'base'
+    
+    @api.model
+    def load(self, fields, data):
```
Same change needed as for the other branch

tfrancoi commented on import_metadata/models/model.py:
```diff
@@ -13,7 +13,7 @@ def _write_metada(self, vals):
             generate the query and execute it to upgrade metadata value directly in SQL since it's the only way to update them.
         """
-        if self.env.context.get('write_metadata') and self.env.uid == 1:
+        if self.env.context.get('write_metadata') and self.env.uid == 2:
```
Use hasgroup('technical settings') this is more generic.
Only for v12

jad-odoo committed:
[IMP] Module import_metadata: allowed for all members of group_system.
Module defer_fields_computation: remove the useless call to search
method + cosmetic.
